### PR TITLE
Feature/allow custom url example app

### PR DESCRIPTION
**Pull request by JiceDessaint**
Add some textfield in example app, allowing user to enter his own url to test.

### CODE REVIEW

File: Example/Main.storyboard [hunk 1]
```diff
@@ -179,8 +179,36 @@
                                             </button>
                                         </subviews>
                                     </stackView>
-                                    <stackView opaque="NO" contentMode="scaleToFill" distribution="fillProportionally" alignment="center" spacing="2" translatesAutoresizingMaskIntoConstraints="NO" id="fhj-ml-7RV" userLabel="Failed Format">
+                                    <stackView opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="AMx-Sx-0V7">
                                         <rect key="frame" x="0.0" y="114" width="355" height="36"/>
+                                        <subviews>
+                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Use Live Custom URL" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="2kL-gu-7N4">
```
Comment (raphrel) [inline Example/Main.storyboard]: can you update to "Use Live Stream Custom URL" to reflect button's name ?

Comment (JiceDessaint) [reply Example/Main.storyboard]: Ok

**Comment by JiceDessaint** [comment]
> Changelog is missing

Done

**Comment by JiceDessaint** [comment]
> Fatal error when you try to load or play emtpy custom url.
> An alert may be preferable at this point.

Done

**Comment by JiceDessaint** [comment]
> It should be "nice" to dismiss keyboard when resign from the textfield

Done

**Comment by raphrel** [comment]
Load Live Stream should load the custom Live Stream URL if set.

**Pull request merged**